.github/ISSUE_TEMPLATE/Plugin_Issue.md

```diff
@@ -5,24 +5,27 @@ about: Create a plugin issue report
 
 <!--
 Thanks for reporting a plugin issue!
+USE THE TEMPLATE, otherwise, your plugin issue may be rejected.
 
+First, see the contribution guidelines:
+https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#contributing-to-streamlink
 
-Please read the contribution guidelines (https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#contributing-to-streamlink) first!
-
-Also check the list of open and closed plugin requests before making a new request! (https://github.com/streamlink/streamlink/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22plugin+request%22+)
-
-Please fill out the following template. Be as detailed as possible.
+Also check the list of open and closed plugin issues:
+https://github.com/streamlink/streamlink/issues?q=is%3Aissue+label%3A%22plugin+issue%22
 
 Please see the text preview to avoid unnecessary formatting errors.
 -->
 
+
 ## Plugin Issue
 
-- [ ] This is a plugin issue and I have read the contribution guidelines. <!-- Replace the space with an x to check the box: [x] -->
+<!-- Replace [ ] with [x] in order to check the box -->
+- [ ] This is a plugin issue and I have read the contribution guidelines.
+
 
 ### Description
 
-<!-- Explain the plugin issue as clearly as you can. Which plugin is causing isuses? what exactly is the issue? Etc. -->
+<!-- Explain the plugin issue as detailed as you can. -->
```

Comment:
Similarly to above the grammar here is weird. I think my proposal from earlier would resolve this, otherwise we can do something else.